test/release/examples/primers/iterators.chpl

```diff
@@ -52,30 +52,37 @@ writeln("...");
 writeln();
 
 //
-// This example uses zipper iteration to iterate over the unbounded range ``1..``
-// and the fibonacci iterator with ``n`` set to ten. Ranges, as well as arrays
-// and domains, can be used as iterators in loops.
+// This example uses zippered iteration to iterate over the fibonacci iterator
+// with ``n`` set to ten and the unbounded range ``1..``. Ranges, as well as
+// arrays and domains, can be used as iterators in loops.
 //
-// Zipper iteration means that each iterator is advanced to its next yield
-// and the two yielded values are combined into a tuple.
+// Zippered iteration means that each iterator is advanced to its next yield
+// and the yielded values are combined into a tuple.
 //
-// A zippered loop can have a single index variable, which will be a tuple,
-// or a tuple of variables like ``(i, j)``, each of which is initialized
-// with the value yielded by the corresponding iterator.
+// A zippered loop can have a single index variable, which will be a tuple
+// of the yielded values; or a tuple of variables like ``(i, j)``, each of
+// which is initialized with the value yielded by the corresponding iterator.
+//
+// In zippered loops, the first iterand controls the behavior of the loop,
+// and is referred to as the "leader".  Subsequent iterands must
+// each either match the number of iterations or else be unbounded, as in
+// this example.  When iterating over multidimensional domains and arrays
+// in a zippered iteration, they must match not only in terms of size, but
+// also shape.
```

Comment:
I suggest explaining what "shape" is, ex.: `... also shape, i.e., the size in each dimension.`

Reply:
Done in https://github.com/chapel-lang/chapel/pull/20891/commits/368dc78d91ab808b363b6f10dcc8f7d163ea885a